Client/core/DXHook/CProxyDirect3D9.cpp

```diff
@@ -172,6 +175,23 @@ HRESULT CProxyDirect3D9::CreateDevice(UINT Adapter, D3DDEVTYPE DeviceType, HWND
     BOOL darkTitleBar = GetSystemRegistryValue((uint)HKEY_CURRENT_USER, "Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize", "AppsUseLightTheme") == "\x0";
     DwmSetWindowAttribute(hFocusWindow, DWMWA_USE_IMMERSIVE_DARK_MODE, &darkTitleBar, sizeof(darkTitleBar));
 
+    // Update icon
+    if (HICON icon = LoadIcon(g_hModule, MAKEINTRESOURCE(IDI_ICON1)))
+    {
+        const auto paramIcon = reinterpret_cast<LPARAM>(icon);
+        for (const WPARAM size : {ICON_SMALL, ICON_BIG})
+        {
+            SendMessage(hFocusWindow, WM_SETICON, size, paramIcon);
+        }
```

Comment:
I don't really mind this, but it's overkill for you could've just duplicated the SendMessage line

Reply:
Is it required?